src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0];
+
+  const isActive = id => id === selectedTabId;
+
+  const handleTabClick = (id, index) => {
+    if (!isActive(id)) {
+      onTabSelected(tabs[index]);
+    }
+  };
```

Comment:
no needs to use index
```suggestion
  const handleTabClick = (tab) => () => {
    if (!isActive(tab.id)) {
      onTabSelected(tab);
    }
  };
```